examples/How_to_call_functions_with_chat_models.ipynb

```diff
@@ -86,21 +86,18 @@
    "source": [
     "@retry(wait=wait_random_exponential(multiplier=1, max=40), stop=stop_after_attempt(3))\n",
     "def chat_completion_request(messages, tools=None, tool_choice=None, model=GPT_MODEL):\n",
```

Comment:
Actually, now that I'm reading through this, with the SDK this whole method is irrelevant. We could replace all calls to this method with direct calls to the SDK

Reply:
@simonpfish - Apologies for the late followup here. Are you suggesting we remove this section and call the client.chat.completions.create for each example? I think there is still some value here in cleaning up the responses in a separate function rather than doing it in each section.

Reply:
We're cleaning up the messages and formatting it prior to passing it to the SDK. Wouldn't all of that logic would have to be duplicated across each example?

Reply:
Not really, the SDK handles all of this under the hood